onnx2torch/node_converters/roialign.py

```diff
@@ -23,30 +26,69 @@ def __init__(
             spatial_scale: float = 1.0,
     ):
         super().__init__()
-
         if mode != 'avg':
             raise NotImplementedError(f'"{mode}" roi align mode is not implemented.')
 
-        self._output_size = (output_height, output_width)
+        self._output_height = output_height
+        self._output_width = output_width
         self._sampling_ratio = sampling_ratio
         self._spatial_scale = spatial_scale
 
-    def forward(
-            self,
+    @staticmethod
+    def _do_forward(
             input_tensor: torch.Tensor,
             rois: torch.Tensor,
             batch_indices: torch.Tensor,
+            output_size: Tuple[int, int],
+            spatial_scale: float,
+            sampling_ratio: int,
     ) -> torch.Tensor:
-        batched_rois = torch.concat([batch_indices.unsqueeze(1).to(rois.dtype), rois], dim=1)
+
+        batch_indices = batch_indices.unsqueeze(1).to(rois.dtype)
+        batched_rois = torch.concat([batch_indices, rois], dim=1)
 
         return roi_align(
             input=input_tensor,
             boxes=batched_rois,
-            output_size=self._output_size,
+            output_size=output_size,
+            spatial_scale=spatial_scale,
+            sampling_ratio=sampling_ratio,
+            aligned=False,
+        )
+
+    def forward(
+            self,
+            input_tensor: torch.Tensor,
+            rois: torch.Tensor,
+            batch_indices: torch.Tensor,
+
+    ) -> torch.Tensor:
+
+        output = self._do_forward(
+            input_tensor=input_tensor,
+            rois=rois,
+            batch_indices=batch_indices,
+            output_size=(self._output_height, self._output_width),
             spatial_scale=self._spatial_scale,
             sampling_ratio=self._sampling_ratio,
-            aligned=False,
         )
+        if torch.onnx.is_in_onnx_export():
+            args = [input_tensor, rois, batch_indices]
+            kwargs = {
+                'output_height' : self._output_height, 
+                'output_width' : self._output_width, 
+                'sampling_ratio' : self._sampling_ratio, 
+                'spatial_scale' : self._spatial_scale
+            }
```

Comment:
1) It is useless kwargs, which will be silently ignored
2) names of attributes must contain type suffix

Reply:
Kwargs are removed